examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -4915,10 +4915,10 @@
               "side": "server",
               "type": "TimeSourceEnum",
               "included": 1,
-              "storageOption": "RAM",
+              "storageOption": "External",
```

Comment:
This is from mTimeSource now, right? So that's RAM.

Reply:
I don't think so. RAM means it is using attribute storage. Here it is using AAI and we don't need the accessors as well.